scripts/checks.sh

```diff
@@ -475,7 +475,7 @@ function test_container_engine_responding() {
 function check_supervisor()
 {
 	# TODO: grab healthcheck results here
-	if ! (${TIMEOUT_CMD} ${ENG} ps | grep -q resin_supervisor) 2> /dev/null; then
+	if ! (${TIMEOUT_CMD} ${ENG} ps | grep -qE 'balena_supervisor|resin_supervisor') 2> /dev/null; then
```

Comment:
```suggestion
	if ! (${TIMEOUT_CMD} ${ENG} ps --filter "name=resin_supervisor" --filter "name=balena_supervisor" -q) 2> /dev/null; then
```

while we're standardizing this, might as well get it here too